### PR TITLE
Remove Ownable and change authorizable

### PR DESCRIPTION
**Pull request by berndbohmeier**
Authorizable now allows only to setup an global authorized address and
init and additionally allows every user to add authorized addresses for
himself.

### CODE REVIEW

File: contracts/tokens/UnwETH.sol [hunk 1]
```diff
@@ -36,6 +35,13 @@ contract UnwEth is Authorizable, Destructable {
         deposit();
     }
 
+    // Todo Remove
+    function addAuthorizedAddress(address target)
+        public
```
Comment (cducrest) [inline contracts/tokens/UnwETH.sol]: Is that normal?

Comment (berndbohmeier) [reply contracts/tokens/UnwETH.sol]: This is not good if we actually deploy this token. As we currently not plan to do that, I did not care enough to solve it properly

File: contracts/lib/Authorizable.sol [hunk 2]
```diff
@@ -1,72 +1,51 @@
...
+    event GlobalAuthorizedAddressRemove(address indexed target);
 
-    event LogAuthorizedAddressAdded(address indexed target, address indexed caller);
-    event LogAuthorizedAddressRemoved(address indexed target, address indexed caller);
-
-    /*
-     * Public functions
-     */
+    event AuthorizedAddressAdd(address indexed target, address indexed sender);
+    event AuthorizedAddressRemove(address indexed target, address indexed sender);
```
Comment (cducrest) [inline contracts/lib/Authorizable.sol]: I don't really like the names of the args, but I can't think of better. What about renaming `target` to `authorized` (and maybe `sender` to `somethingElse`)

File: contracts/lib/Authorizable.sol [hunk 3]
```diff
@@ -1,72 +1,51 @@
...
     /// @dev Removes authorizion of an address.
     /// @param target Address to remove authorization from.
     function removeAuthorizedAddress(address target)
         public
-        onlyOwner
-        targetAuthorized(target)
     {
-        delete authorized[target];
-        emit LogAuthorizedAddressRemoved(target, msg.sender);
+        delete authorized[msg.sender][target];
```
Comment (cducrest) [inline contracts/lib/Authorizable.sol]: Should we `require(authorized[msg.sender][target]);`?

File: contracts/lib/Authorizable.sol (hunk 2 above)
Comment (berndbohmeier) [reply contracts/lib/Authorizable.sol]: Yes, target was taken from the original contract. authorized is ok. I don't have a better name for the other argument. They used caller.

File: contracts/lib/Authorizable.sol (hunk 3 above)
Comment (berndbohmeier) [reply contracts/lib/Authorizable.sol]: Yes, good catch. Same for global

**Pull request merged**